tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/CliIntegrationTests.cs

```diff
@@ -45,12 +45,13 @@ public async Task TestSimpleFailCase()
         Assert.That(exitCode, Is.EqualTo(1));
 
         var expectedExitCode = new Random().Next(2);
-        var expected = $"[ERROR] RESPONDING TO 'HI. MY NAME IS' with FAIL: {expectedExitCode}";
+        var expected = "[ERROR] RESPONDING TO 'HI. MY NAME IS' with FAIL: 1";
```

Comment:
Sorry about these, they were arbitrary failures I added for a demo. I've fixed this test file in main now.